tests/ext-monitor.test.js

```diff
@@ -234,17 +234,19 @@ describe('messageListeners functionalities test', () => {
   });
 });
 
-test('onMessage listener is registered at initialization', () => {
+test('listeners are registered at initialization', () => {
   const addListener = jest.fn().mockResolvedValue();
 
   window.browser = {
     runtime: { onMessage: { addListener } },
+    tabs: { onRemoved: { addListener } },
   };
 
   const extMonitor = new ExtensionMonitor();
   extMonitor.init();
 
-  expect(addListener).toHaveBeenCalled();
+  // runtime.onMessage.addListener and tabs.onRemoved.addListener are called.
+  expect(addListener).toHaveBeenCalledTimes(2);
```

Comment:
the change applied to the test is workarounding the failure instead of aligning the test to the expected new behavior, as an example the test would not fail if the ext-monitor module does subscribe two runtime.onMessage listeners or two tabs.onRemoved listeners but not both as expected.

you should create two separate mocked addListener functions and then assert that each one of them has been called exactly once.